articles/managing-labels-in-fleet.md

```diff
@@ -2,99 +2,23 @@
 
 ![Managing labels in Fleet](../website/assets/images/articles/managing-labels-in-fleet-1600x900@2x.png)
 
+In Fleet you can labels (tags), to scope [software](https://fleetdm.com/guides/deploy-software-packages), [policies](https://fleetdm.com/securing/what-are-fleet-policies), [queries](https://fleetdm.com/guides/queries), and [configuration profiles](https://fleetdm.com/guides/custom-os-settings) to specific hosts. In addition, you can use labels to create a custom filtered view of your hosts.
```

Comment:
```suggestion
In Fleet, you can add labels (tags) to specific hosts to scope [software](https://fleetdm.com/guides/deploy-software-packages), [policies](https://fleetdm.com/securing/what-are-fleet-policies), [queries](https://fleetdm.com/guides/queries), and [configuration profiles](https://fleetdm.com/guides/custom-os-settings). You can also use labels to create a custom, filtered view of your hosts.
```

Reply:
Rephased for ease of reading (at least for me). I also removed "(tags)." I feel like tags and labels are more or less the same thing.